src/main/java/gift/product/model/Option.java

```diff
@@ -56,4 +56,7 @@ public Product getProduct() {
     public boolean isSameName(String name) {
         return this.name.equals(name);
     }
+    public void subtractQuantity(int quantity) {
+        this.quantity -= quantity;
+    }
```

Comment:
```suggestion
    public void subtractQuantity(int amountToSubtract) { // input 변수명 수정
    // amountToSubtract 가 음수인지 확인
    if (this.quantity < amountToSubtract) {
        // 에러처리
    }
        this.quantity -= quantity;
    }
```
이런식으로 예외를 처리하는게 어떨까요?